com.alibabainc.DingTalk.metainfo.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.alibabainc.DingTalk</id>
+  <name>DingTalk</name>
+  <project_license>LicenseRef-proprietary</project_license>
+  <developer id="com.dingtalk">
+    <name>DingTalk Alibaba Inc</name>
+  </developer>
+  <summary>DingTalk, Make It Happen</summary>
+  <summary xml:lang="zh-cn">钉钉, 让进步发生!</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <launchable type="desktop-id">com.alibabainc.DingTalk.desktop</launchable>
+  <url type="homepage">https://www.dingtalk.com/</url>
+  <url type="help">https://www.dingtalk.com/qidian/helpCenter/pcPage</url>
+  <description>
+    <p>钉钉是阿里巴巴集团打造的企业级智能移动办公平台，是AI时代的全新工作方式。</p>
+    <p>钉钉整合了AI助理、IM即时沟通、钉钉文档、多维表、视频会议、企业邮箱、Teambition、OA审批、智能人事、考勤等功能。</p>
+    <p>NOTE: This wrapper is not verified by, affiliated with, or supported by Alibaba Inc.</p>
```

Comment:
need an english version of this, you can put the Chinese one under xml:lang https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines#metainfo-translations